cluster/addons/dns/README.md

```diff
@@ -89,8 +89,8 @@ of this yourself.  First, each kubelet needs to run with the following flags
 set:
 
 ```
---cluster_dns=<DNS service ip>
---cluster_domain=<default local domain>
+--cluster-dns=<DNS service ip>
```

Comment:
Double checked and found a second place I was overzealous.  In both cases it is/was because a non-kube/NormalizeUsing program used the same flag name as the ones I was 'sure' about. In this case both the kubelet and kube2sky use `--cluster_dns`.  So I accidentally updated here.  In the other case it was (again a doc for) ginko/e2e.

I'm not sure how to restructure this to make things better/easier.  Someone else just needs to double check all of the changes I think. I'd be happy to break things apart if you think there's a reasonable way to do so....